### PR TITLE
docs *: merge radondb-clickhouse-kubernetes docs #13

### LINKED ISSUE #13: merge radondb-clickhouse-kubernetes repository
target: merge radondb-clickhouse-kubernetes repository

task:
- [x] Move helm charts to this repository
- [x] Integrate documentation from `radondb-clickhouse-kubernetes` into this repository
- [ ] Modify the relevant deployment documentation in KS and elsewhere (if any)
- [ ] Add git pages to this repository
- [ ] Add shell scripts to automatically generate helm charts files from deploy files
- [ ] Modify all docs from source（altinity）
- [ ] Add zh-cn docs

### PR DESCRIPTION
**Pull request by molliezhang**
target: merge radondb-clickhouse-kubernetes  docs

task:
- [x] Integrate documentation from `radondb-clickhouse-kubernetes` into this repository
- [x] Modify the relevant deployment documentation in KS and elsewhere (if any)
- [x] Modify quick-start.md in docs
- [x] Add quick-start.md in zh-cn
- [x] Modify README.md
- [x] Add README_zh.md

### CODE REVIEW

File: docs/zh-cn/quick_start.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+
+```bash
+$ kubectl get pods -n test-clickhouse-operator
+NAME                                 READY   STATUS    RESTARTS   AGE
+clickhouse-operator-5ddc6d858f-drppt 1/1     Running   0          1m
+```
+
+### 从源头创建 Operator
+
+关于如何从源头如何部署 RadonDB ClickHouse Operator，以及如何部署一个 docker 镜像并在 `kubernetes` 中使用，请参考[从源头创建 Operator](../operator_build_from_sources.md)。
```
Comment (sozenh) [inline docs/zh-cn/quick_start.md]: 从源码构建

Comment (molliezhang) [reply docs/zh-cn/quick_start.md]: OK

**Comment by su-houzhen** [comment]
LGTM

**Review by sozenh — APPROVED** [review]
LGTM

**Pull request merged**